README.md

```diff
@@ -1,33 +1,81 @@
+**Delivery mindset**
+
+We believe in being pragmatic and moving fast, spending our energy on the right things, shipping code and then rapidly iterating on it. 
+
+_Behaviours_:
+- We simplify our work, focusing on getting it to the user as early as possible.
+- We practice [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery)
+- We are trying to [make the change easy, before the easy change](https://twitter.com/KentBeck/status/250733358307500032).   
+- We aim to solve issues rapidly prioritising them based on impact they may have.
+
+**Inclusiveness & Respect** 
 
 We believe that our culture is important, and we all contribute to it regardless of our role or level.
 
->Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+_Behaviours_:
+ - We ensure to treat everyone we encounter with respect independently of their skillsets and levels of ability
+ - Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+ - We respect the opinion of others and are gracious in our interactions, whether meeting physically or digitally, whether friend or stranger, whether employee, customer or commenter.
+ - We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
+ - We value and respect all differences in all people (seen and unseen). We aspire to inclusive working experiences and an environment that reflects the audience we serve, where our people have equal access to career development opportunities, their voices are heard and can contribute to our future. We aim to hire, retain and advance people from underrepresented groups.
```

Comment:
@johnduffell what do you think of this suggestion, regarding your previous comment?

Reply:
great.  if we are happy to strengthen it to "does happen" rather than "can" that would be even better
```suggestion
 - We value and respect all differences in all people (seen and unseen). We aspire to inclusive working experiences and an environment that reflects the audience we serve, where our people take up career development opportunities equally, their voices are heard and contribute to our future. We aim to hire, retain and advance people from underrepresented groups.
```